packages/block-library/src/page-list/edit.js

```diff
@@ -220,7 +220,11 @@ export default function PageListEdit( {
 			const hasChildren = pagesByParentId.has( page.id );
 			const pageProps = {
 				id: page.id,
-				label: page.title?.rendered,
+				label:
+					// translators: displayed when a page has an empty title.
+					page.title?.rendered !== ''
```

Comment:
```suggestion
					page.title?.rendered?.trim() !== ''
```

Let's check for whitespace.

Reply:
Thanks!